maitake/src/scheduler.rs

```diff
@@ -102,19 +115,18 @@ impl Core {
     /// Chosen by fair dice roll, guaranteed to be random.
     const DEFAULT_TICK_SIZE: usize = 256;
 
+    #[cfg(feature = "alloc")]
     fn new() -> Self {
         let stub_task = TaskRef::new(Stub, Stub);
         Self {
             run_queue: MpscQueue::new_with_stub(test_dbg!(stub_task)),
         }
     }
 
-    fn spawn_static(&'static self, future: impl Future) {
-        self.schedule(TaskRef::new(self, future));
-    }
-
-    fn spawn_arc(this: &Arc<Self>, future: impl Future) {
-        this.schedule(TaskRef::new(this.clone(), future));
+    const unsafe fn new_with_static_stub(stub: &'static Header) -> Self {
```

Comment:
needs to be
```suggestion
    #[cfg(not(loom))]
    const unsafe fn new_with_static_stub(stub: &'static Header) -> Self {
```
since `loom`'s `AtomicPtr::new` can't be `const`: https://github.com/hawkw/mycelium/runs/6677548757?check_suite_focus=true